application-development/helloworld-springboot-auth/.idea/modules.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
an we remove this file?

I don't think we should be committing any IDE configuration files.
thanks